install.sh

```diff
@@ -383,7 +383,7 @@ test_git() {
   fi
 
   local git_version_output
-  git_version_output="$("$1" --version 2>/dev/null)"
+  git_version_output="$("$1" --version 2>/dev/null | awk '{ print $1 " " $2 " " $3 }')"
   version_ge "$(major_minor "${git_version_output##* }")" "$(major_minor "${REQUIRED_GIT_VERSION}")"
```

Comment:
```suggestion
  git_version_output="$("$1" --version 2>/dev/null)"
  if [[ "$git_version_output" =~ "git version "([^ ]*).* ]]; then
    version_ge "$(major_minor "${BASH_REMATCH[1]}")" "$(major_minor "${REQUIRED_GIT_VERSION}")"
  else
    abort "Unexpected Git version string '${git_version_output}' reported"
  fi
```

Reply:
> Matching the semver version with a regex and then passing it into sort -V seems more stable and correct but thats for another day.

As I said I would like to delay the version check refactor for a new pr and keep this about making sure the installer works on all systems again. But if you have hard feelings on this I can click the commit suggestion button.

Reply:
The change I'm requesting actually _matches the version string prefix_, rather than just assuming it's "git version". This is additional assurance that the installer is reading the right information.

Reply:
Yes I understand the code ._. as I said I planned to do the same in a new pr. Since I want to keep this hot fix as simple as possible to ensure that the known bug is fixed as fast as possible. But since you seem to be insisting on this change I made sure its working as expected and tested it locally and committed it :slightly_smiling_face: